app/controllers/courses_controller.rb

```diff
@@ -14,4 +14,17 @@ def show
   rescue JsonApiClient::Errors::NotFound
     render file: "errors/not_found", status: :not_found
   end
+
+  def apply
+    course = Course
+      .includes(:provider)
+      .where(recruitment_cycle_year: Settings.current_cycle)
+      .where(provider_code: params[:provider_code])
+      .find(params[:course_code])
+      .first
+
+    Rails.logger.info("Course apply conversion. Provider: #{course.provider.provider_code}. Course: #{course.course_code}")
+
+    redirect_to "https://www.apply-for-teacher-training.education.gov.uk/candidate/apply?providerCode=#{course.provider.provider_code}&courseCode=#{course.course_code}"
```

Comment:
can we stick the url in to settings.

Reply:
it's only used here and the spec so i'm for with it for now